std/getopt.d

```diff
@@ -859,12 +859,12 @@ private bool handleOption(R)(string option, R receiver, ref string[] args,
             if (val.length)
             {
                 *receiver = to!(typeof(*receiver))(val);
-                break;
+                continue;
             }
 
             // no argument means set it to true
             *receiver = true;
-            break;
+            continue;
```

Comment:
I see, you are correct. The if-clauses preceding this one use 'break' or 'continue'. I assume 'continue' with the intent that further if-tests in the loop not be executed. I assume this if-clause was following suit when it incorrectly used 'break'. However, the subsequent if-clauses do not use either 'break' or 'continue', as each if-clause block is actually the last clause executed in the loop. I didn't catch that part. I'll change it.